packages/webcomponents/src/api/Document.ts

```diff
@@ -0,0 +1,133 @@
+
+export interface IDocument {
+  business_id: string;
+  created_at: string;
+  description: string | null;
+  document_type: string;
+  file_name: string;
+  file_type: string;
+  id: string;
+  identity_id: string;
+  metadata: any;
+  platform_account_id: string;
+  presigned_url: string | null;
+  status: string;
+  updated_at: string;
+}
+
+export class Document implements IDocument {
+  public business_id: string;
+  public created_at: string;
+  public description: string | null;
+  public document_type: string;
```

Comment:
I think this could be

```suggestion
  public document_type: EntityDocumentType;
```

Reply:
Hmmm good catch! 

I actually wonder if we should just remove that entire class from the code for now - the `Document` class that's declared in the top of the file is not used anywhere else - that code used to live in the Business.ts file but even then was never used anywhere. 

The `IDocument` interface can stay, since `Business.ts` imports it and uses it to type its `document` property. 

What do you think? Keep or remove this class and its constructor? 

@TiagoFuelber

Reply:
I didn't noticed it was not being used. In that case I think it's alright to remove the class. This will keep our code cleaner. Good catch!